TM-UI-NEXT from develop to master Code push regarding CI Build Configurations

bhpratibha opened the pull request:
TM-UI-NEXT from develop to master Code push

Commit:
registration login forget pass reset pass security service service point changes

Commit:
github workflows

Commit:
json files added

Commit:
lint issues resolved

Commit:
worklist changes

Commit:
visit details, vitals, examination, anc, pnce, search and register changes

Commit:
history, idrs, ncd screening, quick consult changes

Commit:
doctor screen changes

Commit:
changes in core components and quick consult changes

Commit:
Table width CSS chnages

Commit:
nurse and doctor changes

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI-NEXT into develop

Commit:
doctor update changes

Commit:
minor changes in popup

Commit:
css changes in all screens

Commit:
pharmcist routing to inventory changes

Commit:
nurse doctor changes

Commit:
enabling functionality for the complaint chief

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI-NEXT into develop

Commit:
fixed issues

Commit:
nurse save changes

Commit:
minor changes

Commit:
resolved the issues for the past-obsteric-history

Commit:
nurse doctor changes

Commit:
issue fixes

Commit:
minor changes n css

Commit:
CSS changes

Commit:
ci build configurations changes

Commit:
ci build configurations changes

Commit:
ci environmentfile changes

Commit:
Merge branch 'master' of https://github.com/PSMRI/TM-UI-NEXT into develop

drtechie commented on pom.xml:
```diff
@@ -0,0 +1,199 @@
+              <executable>npm</executable>
+              <arguments>
+                <argument>install</argument>
+                <argument>--legacy-peer-deps</argument>
```
Same comment as other repos. Not needed. please check.

bhpratibha replied:
Ignoring this comment as per MMU-UI-NEXT Comments

drtechie commented:
CI template file seems to be missing.

Commit:
fixed PR comments

bhpratibha commented:
> CI template file seems to be missing.

Added please check

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI-NEXT into develop

Commit:
prescription quantity change

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI-NEXT into develop

Commit:
nurse-doctor changes

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI-NEXT into develop

Commit:
CSS changes

Commit:
history and abha changes

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI-NEXT into develop

Commit:
css and enabling and disabling changes

bhpratibha committed:
added fix for the on change of visit category refresh the history pages

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI-NEXT into develop

Commit:
customization changes

Commit:
missed changes

Commit:
service point extra details save for customization

Commit:
bug fixes

Commit:
minor change in workarea

Commit:
added .gitmodules folder

Commit:
app header changes

Commit:
json changes and environment url change

NKOKKILIGADDA87 committed:
CSS CHNAGES

NKOKKILIGADDA87 committed:
CSS CHNAGES

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

devikasuresh20 committed:
Changes for patching height on patient revisit

ravishanigarapu committed:
Merge pull request #15 from devikasuresh20/develop

AMM-712 Changes for patching height on patient revisit

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
CSS Changes

devikasuresh20 committed:
Changes done for Revisit filteration issue

devikasuresh20 committed:
Resolved conflicts

ravishanigarapu committed:
Merge pull request #16 from devikasuresh20/develop

AMM-683 - Revisit Search Issue

pavangsk committed:
Update environment.ci.ts.template

Removed duplicate variables

NKOKKILIGADDA87 committed:
CSS Changes

devikasuresh20 committed:
Changes for adding Revisit & Refer section for Gen OPD QC

devikasuresh20 committed:
Resolved merge conlflicts

NKOKKILIGADDA87 committed:
CSS chnages

devikasuresh20 committed:
Added code improvements according to JS and also resolved prettier eslint related issues

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
Resolved merge conflicts

devikasuresh20 committed:
Minor change in workarea

Commit:
date format issue fixed

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
Changes done for removing covid vaccination status

devikasuresh20 committed:
Resolved merge conflicts

ravishanigarapu committed:
Merge pull request #18 from devikasuresh20/develop

AMM-754 Addition of Referral section in General OPD QC and AMM-772 Removal of Covid Vaccination status section

Commit:
Resolved Merged conflicts and merged customization to develop

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI into develop

Commit:
added refer date issue resolved

NKOKKILIGADDA87 committed:
CSS chnages

NKOKKILIGADDA87 committed:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI into develop

NKOKKILIGADDA87 committed:
CSS chnages

NKOKKILIGADDA87 committed:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI into develop

NKOKKILIGADDA87 committed:
CSS chnages

NKOKKILIGADDA87 committed:
CSS chnages

Commit:
change in css for radio button for tablet view

Commit:
CSS responsive changes

pavangsk committed:
CSS responsive changes

raj9885 committed:
CSS responsive Chnages

raj9885 committed:
CSS responsive Chnages

NKOKKILIGADDA87 committed:
CSS Changes

Commit:
bio changes

Commit:
merge conflicts resolved

Commit:
changes in health-id-display-modal for tm

Commit:
Change in code for visitCode for local storage

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

NKOKKILIGADDA87 committed:
CSS changes

Commit:
Changes for history Past Obstetric History in  PNC visit category for tm

Commit:
Added common-ui configuration in git hub workflows

NKOKKILIGADDA87 committed:
CSS Changes

NKOKKILIGADDA87 committed:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI into develop

NKOKKILIGADDA87 committed:
CSS Changes

Commit:
Modified README file for submodule common-ui

anandamideShakyan committed:
[33] - Updated  outputPath in angular.json    

Updated outputPath in angular.json to fix the Forbidden error during wildfly deployment.

drtechie committed:
Merge pull request #24 from anandamideShakyan/develop

[33] - Updated  outputPath in angular.json

NKOKKILIGADDA87 committed:
es lint errors fix

NKOKKILIGADDA87 committed:
upload file cancel button fix

NKOKKILIGADDA87 committed:
ng build CSS files syntex errors fix

NKOKKILIGADDA87 committed:
common_UI deployement

NKOKKILIGADDA87 committed:
readme file changes  and add nvmrc file

NKOKKILIGADDA87 committed:
swaasa issue

NKOKKILIGADDA87 committed:
comman UI wardeployement

Commit:
Change in interceptor service for Swasaa

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI into develop

NKOKKILIGADDA87 committed:
Change in interceptor service for Swasaa

NKOKKILIGADDA87 committed:
comman UI wardeployement

anandamideShakyan committed:
[33] - Update environment.ci.ts.template

anandamideShakyan committed:
[33] - Update environment.ci.ts.template

anandamideShakyan committed:
Update environment.ci.ts.template

anandamideShakyan committed:
Update environment.ci.ts.template

NKOKKILIGADDA87 committed:
comman UI wardeployement

NKOKKILIGADDA87 committed:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI into develop

drtechie committed:
Merge pull request #28 from anandamideShakyan/develop

[33] - Update environment.ci.ts.template

NKOKKILIGADDA87 committed:
warfile deployement

devikasuresh20 committed:
Added mandatory condition for Travel History section

devikasuresh20 committed:
Added mandatory condition for Travel History section (#30)

devikasuresh20 committed:
URL changes using jboss context-root

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
Minor change in angular.json

devikasuresh20 committed:
Minor change

devikasuresh20 committed:
Merge pull request #31 from devikasuresh20/develop

AMM-975 Remove "ui-next" from URL

devikasuresh20 committed:
Workarea related change in nurse module

devikasuresh20 committed:
Workarea related change in nurse module (#35)

devikasuresh20 committed:
ANC menstrual history changes

devikasuresh20 committed:
AMM-1013 ANC menstrual history changes (#36)

* Workarea related change in nurse module

* ANC menstrual history changes

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
File upload changes

devikasuresh20 committed:
Minor changes in file upload

devikasuresh20 committed:
AMM-941 & AMM-964 File upload changes (#37)

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Minor changes in file upload

gupta-soham committed:
Add ESLint and NPM Audit workflow

drtechie committed:
Merge pull request #38 from gupta-soham/feat/add-lint-audit-workflow

Add ESLint and NPM Audit workflow

devikasuresh20 committed:
Diagnosis changes

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
AMM-1009 Diagnosis changes (#39)

* Workarea related change in nurse module

* ANC menstrual history changes

* File upload changes

* Minor changes in file upload

* Diagnosis changes

devikasuresh20 committed:
Merge branch 'PSMRI:develop' into develop

devikasuresh20 committed:
CSS changes for Schedule for TM Screen

devikasuresh20 committed:
Minor changes

devikasuresh20 committed:
Minor changes

devikasuresh20 committed:
Merge pull request #42 from devikasuresh20/develop

AMM-1053 Schedule for TM changes

devikasuresh20 committed:
Added scroll bar for Schedule TM pop up (#43)

devikasuresh20 committed:
AMM-1055  Family History Diabetes Mellitus info message changes (#44)

* Added scroll bar for Schedule TM pop up

* NCD Screening family historychanges

* Minor changes

devikasuresh20 committed:
AMM-1063 File upload changes (#45)

* Added scroll bar for Schedule TM pop up

* NCD Screening family historychanges

* Minor changes

* File upload patching changes in doctor

devikasuresh20 committed:
AMM-1008 Previous ben visit details changes (#46)

* Added scroll bar for Schedule TM pop up

* NCD Screening family historychanges

* Minor changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

NKOKKILIGADDA87 committed:
tm wasa first commit

NKOKKILIGADDA87 committed:
tm wasa second commit

NKOKKILIGADDA87 committed:
tm wasa third commit

NKOKKILIGADDA87 committed:
tm wasa forth commit

NKOKKILIGADDA87 committed:
tm wasa fifth commit

NKOKKILIGADDA87 committed:
tm wasa sixth commit

NKOKKILIGADDA87 committed:
tm wasa seventh commit

NKOKKILIGADDA87 committed:
tm wasa eighth commit

NKOKKILIGADDA87 committed:
tm wasa ninth commit

NKOKKILIGADDA87 committed:
tm wasa tenth commit

NKOKKILIGADDA87 committed:
ngx cookie service version change

devikasuresh20 committed:
AMM-1106 Systolic and Diastolic mandatory changes for NCD screening and GenOPD (#51)

* Added scroll bar for Schedule TM pop up

* NCD Screening family historychanges

* Minor changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* Systolic and Diastolic BP changes

sandipkarmakar3 committed:
sonar issue fix with session service path change and encryption key location change

Commit:
added Urls for common ui

Commit:
Merge branch 'develop' of https://github.com/PSMRI/TM-UI into develop

Commit:
Abdm facility and version 3 changes

helenKaryamsetty committed:
Update minor change in workarea.component.ts

devikasuresh20 committed:
Merge pull request #52 from helenKaryamsetty/develop

Abdm facility and version 3 changes

Commit:
facility changes

Commit:
Merge branch 'develop' of https://github.com/helenKaryamsetty/TM-UI-New into develop

helenKaryamsetty committed:
Merge branch 'PSMRI:develop' into develop

Commit:
facility logic correction

Commit:
Merge branch 'develop' of https://github.com/helenKaryamsetty/TM-UI-New into develop

NKOKKILIGADDA87 committed:
remove JWT tokken setItem

devikasuresh20 committed:
Merge pull request #53 from helenKaryamsetty/develop

abdm facility comparision correction

NKOKKILIGADDA87 committed:
remove EncKey

NKOKKILIGADDA87 committed:
diagnosis value feching

NKOKKILIGADDA87 committed:
sessionStorageEncKey value emity

NKOKKILIGADDA87 committed:
diagnosis value feching

sandipkarmakar3 committed:
minor TM fixes

sandipkarmakar3 committed:
minor TM fixes

Sneha6003 committed:
Update README.md

sandipkarmakar3 committed:
ready to merge with develop

sandipkarmakar3 committed:
ready to merge with develop

sandipkarmakar3 committed:
-wasa removed

Asp-irin committed:
assigned port 4023 and updated all the endpoints in the env

Asp-irin committed:
assigned port 4023 and updated all the endpoints in the env

Asp-irin committed:
added env.ts to gitignore

drtechie committed:
Merge pull request #54 from Asp-irin/develop

Assigned port 4203 and updated all the endpoints in the env for local development

sandipkarmakar3 committed:
Devika coments resolved

devikasuresh20 committed:
Merge pull request #48 from PSMRI/tm-wasa

Tm wasa

drtechie committed:
Add LICENSE and CODE_OF_CONDUCT

sandipkarmakar3 committed:
minor fix

devikasuresh20 committed:
Merge pull request #55 from PSMRI/minorFix

minor fix

devikasuresh20 committed:
Previous Ben Visit Details Changes (#56)

* Added scroll bar for Schedule TM pop up

* NCD Screening family historychanges

* Minor changes

* File upload patching changes in doctor

* Added scroll for previous ben visit details

* Minor changes

* Systolic and Diastolic BP changes

* previous visit details changes

Zapper9982 committed:
fix: Made necessary changes for build to pass

Zapper9982 committed:
fix:  added filereplacement

Zapper9982 committed:
fix: updated .eslintrc.json

Zapper9982 committed:
fix: updated angular.json

Zapper9982 committed:
fix: discaded package-prod.yml and updated package.yml

Zapper9982 committed:
fix: changed typo to tm-ui

drtechie committed:
Merge pull request #57 from Zapper9982/bugfix/issue-51

fix: Made necessary changes for build to pass

yashpatil641 committed:
feat(commit-config): add conventional commit standards with commitlint, husky and commitizen

Implement standardized commit message format using Commitlint, Husky, and Commitizen.

drtechie committed:
Merge pull request #58 from yashpatil641/v2

feat(commit-config): add conventional commit standards with commitlint, husky and commitizen

srinuatpiramal committed:
Update environment.ci.ts.template

srinuatpiramal committed:
Update environment.dev.ts

srinuatpiramal committed:
Update environment.development.ts

srinuatpiramal committed:
Update environment.development.ts

srinuatpiramal committed:
Update environment.local.ts

srinuatpiramal committed:
Update environment.prod.ts

srinuatpiramal committed:
Update environment.prod.ts

srinuatpiramal committed:
Update environment.test.ts

srinuatpiramal committed:
Update environment.test.ts

srinuatpiramal committed:
fix(auth): adding envs (#61)

Co-authored-by: Srinivas Tirunahari <srinivas_t@piramalfndn.dc>

drtechie committed:
feat: ensure url doesn't have version. upgrade version

drtechie committed:
fix: war file artifact names

drtechie committed:
Merge pull request #63 from PSMRI/build-update

feat: ensure url doesn't have version. upgrade version

5Amogh committed:
feat: VAPT captcha secuirty addition

5Amogh committed:
fix: captcha reset issue fix

5Amogh committed:
Merge pull request #64 from PSMRI/develop-amogh

#AMM-1247- feat: VAPT Captcha implementation

5Amogh committed:
fix: type fix for the enable captcha variable (#65)

snehar-nd committed:
fix: login and forgot pw error response changes (#66)
